tests/phpunit/src/Commands/NewCommandTest.php

```diff
@@ -94,11 +94,6 @@ public function testNewCommand($project): void {
     $this->assertStringContainsString($project, $output);
     $this->assertStringContainsString('New 💧Drupal project created in ' . $this->newProjectDir, $output);
 
-    if ($project === 'drupal/recommended-project') {
-      $composer_json = file_get_contents(Path::join($this->newProjectDir, 'composer.json'));
-      $this->assertStringNotContainsString('web/', $composer_json);
-      $this->assertStringContainsString('docroot/', $composer_json);
```

Comment:
I think we should still assert that we have the expected composer.json in the right place.

Reply:
I don't think there's any value in this test, because we've hardcoded a composer.json in the text fixture, so all we'd testing is the fixture, nothing related to the business logic in the code.